### PR TITLE
Add hypnotube.com xPath scene scraper

### PR DESCRIPTION
**Pull request by Triskelia**
Add scene url scraper for hypnotube.com
Contains mostly Hypnosis, JOIs, PMVs, Feminization, Sissification and Bimbofication videos

Scrapes : 
- Title
- Details
- Image (From curated thumbnail, as shown in the search section of the site)
- Studio
  - Name (poster's username)
  - URL (poster's hypnotube profile url)
- Date
- Tags (From both Tags and Categories sections on the page)

### CODE REVIEW

File: scrapers/Hypnotube.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - hypnotube.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //a[starts-with(@href,'https://hypnotube.com/user/')]
+    scene:
+      Title: //div[@class='item-tr-inner-col inner-col']/h1/text()
+      Details: //meta[@name='description']/@content
+      Image: //meta[@property='og:image']/@content
```
Comment (bnkai) [inline scrapers/Hypnotube.yml:14]: Not working for me (at least with the scenes i checked)
I tried to pull the image from a js that is in the page. It seems to work ok for me.
```yaml
      Image:
        selector: //script[@type="text/javascript" and contains(text(),"twitter:image")]/text()
        postProcess:
          - replace:
              - regex: '.+"twitter:image" content="([^"]+)".+'
                with: $1
```

File: scrapers/Hypnotube.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    url:
+      - hypnotube.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //a[starts-with(@href,'https://hypnotube.com/user/')]
+    scene:
+      Title: //div[@class='item-tr-inner-col inner-col']/h1/text()
+      Details: //meta[@name='description']/@content
```
Comment (bnkai) [inline scrapers/Hypnotube.yml:13]: For longer descriptions details are truncated...
We could get the details from the same script that has the image. Some special chars would need to be replaced though. Is it worth it to check that out?

File: scrapers/Hypnotube.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      Image: //meta[@property='og:image']/@content
+      Studio:
+        Name: $studio/@title
+        URL: $studio/@href
+      Date:
+        selector: //div[@class='stats-container']/ul/li[3]/span[@class='sub-label']/text()
+        postProcess:
+          - replace:
+            - regex: \s\d{2}:\d{2}:\d{2}
+              with: ""
```
Comment (bnkai) [inline scrapers/Hypnotube.yml:23]: a little simpler
```yaml
      Date:
        selector: //li/span[@class="icon i-calendar"]/following-sibling::span
        postProcess:
          - replace:
              - regex: \s.*$
                with: ""
```

File: scrapers/Hypnotube.yml (hunk 1 above)
Comment (Triskelia) [reply scrapers/Hypnotube.yml]: Looks like they changed something on the website. Your method works well. I've added it in my last commit

File: scrapers/Hypnotube.yml (hunk 2 above)
Comment (Triskelia) [reply scrapers/Hypnotube.yml]: Indeed, I should have tested videos with longer description. In my last commit, I went for the actual description in the page (class "main-description"). That way, we don't need to replace special char as we would have to if we were to scrap it from the js.

File: scrapers/Hypnotube.yml (hunk 3 above)
Comment (Triskelia) [reply scrapers/Hypnotube.yml]: Looks good. Added it in my last commit

**Pull request merged**